ci.sh

```diff
@@ -128,6 +135,7 @@ pgrep -f target/debug/parsec >/dev/null
 if [ "$PROVIDER_NAME" = "all" ]; then
     echo "Execute all-providers tests"
     RUST_BACKTRACE=1 cargo test --manifest-path ./e2e_tests/Cargo.toml all_providers
+    RUST_BACKTRACE=1 CONFIG_PATH=$CONFIG_PATH PARSEC_PID=$PARSEC_PID cargo test --manifest-path ./e2e_tests/Cargo.toml config
```

Comment:
I am not really keen on adding more environment variables 😬 I am afraid of seeing this expanded more and more until the line become very long... Specially if they are needed just for one test.
I propose the following instead:
* `CONFIG_PATH` seems to never change and be `$(pwd)/e2e_tests/provider_cfg/tmp_config.toml`? Can not we hardcode that in the test?
* `PARSEC_PID`: our crate generates a binary named `parsec`. I think it is fine referring to the `parsec` process through its name and the `pkill` command.

Reply:
Excellent, good ideas!

Reply:
Done